paper-slider.html

```diff
@@ -568,6 +572,7 @@
         this._w = this.$.sliderBar.offsetWidth;
         this._x = this.ratio * this._w;
         this._startx = this._x || 0;
+        this._knobstartx = this._startx;
```

Comment:
You would have to reset [`e.detail.dx`](https://github.com/PolymerElements/paper-slider/pull/48/files#diff-796ae68652c9d9635b1db7325e829a16R587) to 0 from [line 547](https://github.com/PolymerElements/paper-slider/pull/48/files#diff-796ae68652c9d9635b1db7325e829a16R547), which I think does not make sense.